cmd/http/server.go

```diff
@@ -120,6 +125,11 @@ func (srv *Server) Start() (err error) {
 
 	// Start servicing with listener.
 	if tlsConfig != nil {
+		if err = http2.ConfigureServer(srv.Server, &http2.Server{
```

Comment:
As `http2.ConfigureServer()` requires pointer of `http.Server` here, you could do `&srv.Server` than extending a pointer of `*http.Server`.  Extending pointer doesn't initialize internal fields which requires separate making and assignment like `Server: &http.Server{}`.

Reply:
That I already did @balamurugana in the below code @balamurugana

Reply:
The point is that the change is not required.

Reply:
Sure I can do that way too.